client/src/analysis/AnalysisTypes.res

```diff
@@ -387,6 +387,8 @@ module FourOhFour = {
   }
 }
 
+// TODO: these are used outside of analysis (e.g. in APIWorkers) so the type def
+// should probably move?
 module WorkerState = {
```

Comment:
Happy to leave this for another PR, but `WorkerState` types are used both in "analysis" and during initial load, so I think we should move these defs somewhere